mods/ra/rules/infantry.yaml

```diff
@@ -72,6 +74,16 @@ E1:
 	ProducibleWithLevel:
 		Prerequisites: barracks.upgraded
 
+E1R1:
+	Inherits: E1
+	RenderSprites:
+		Image: E1
+	ProducibleWithLevel:
+		Prerequisites: techlevel.infonly
+		InitialLevels: 1
+	Buildable:
```

Comment:
This leaves them buildable in the sidebar with cheats enabled.  It would be better to use `-Buildable` here to indicate that they really aren't buildable at all.